generic/utils/ras.py.data/drmgr_tool.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
Would it be possible to find the original authors of those scripts as well as licenses (unless it's public domain) and add them in the scripts heading?